src/modules/pf3dTracker/src/pf3dTrackerSupport.cpp

```diff
@@ -160,4 +124,3 @@ void rgbToYuvBinLut(int &R, int &G, int &B, int &YBin, int &UBin, int &VBin, Lut
     if(VBin<0||VBin>7)
         std::cout<<"something's wrong with V: "<<VBin<<" "<<V<<" R= "<<R<<" G= "<<G<<" B= "<<B<<"\n";
```

Comment:
@gsaponaro this is extremely minor but since you already changed most of the `cout` to use `yError` or `yWarning` (in this case), the idea was to use a uniform printout method (change them all)